services/kommander/0.5.2/defaults/cm.yaml

```diff
@@ -107,7 +107,7 @@ data:
           traefik: "20.8.0"
           kubernetes-dashboard: "6.0.0"
           kubecost: "0.33.2"
-          kube-prometheus-stack: "44.2.2"
+          kube-prometheus-stack: "46.8.1"
```

Comment:
😅 Let me cut another release off 44.2.x in the chart so we don't jump major versions again

Reply:
https://github.com/mesosphere/charts/pull/1451